ui/settings: fix sizer option threshold

rr- opened the pull request:
#### Checklist

- [x] I have read the [coding conventions](https://github.com/LostArtefacts/TRX/blob/master/CONTRIBUTING.md#coding-conventions)
- [x] I have added a changelog entry about what my pull request accomplishes, or it is an internal change
- [x] I have added a readme entry about my new feature or OG bug fix, or it is a different change

#### Description

Resolves #3123.

rr- committed:
ui/settings: fix sizer option threshold

Resolves #3123.

aredfan reviewed (approved):
The issue is fixed, thank you. 👍 

I've noticed an anomaly while testing.  If I repeatedly press `F10`, the game will shrink 7 times from maximum to minimum size. If the `Sizer` toggle in Graphics/Rendering is used instead then the game will shrink 6 times in total instead of 7. I'm not sure if it's an issue but thought I'd mention it.

rr- committed:
ui: use 2 decimal places for floats

rr- committed:
tr2/ui/settings: fix sizer inconsistencies

rr- commented:
Thanks @aredfan very good catch. The deltas were different – the UI would change by 0.1, whereas the game would change by 0.08. I opted to make it 0.05 in both cases. To make the UI display the more precise values, I changed it to use 2 decimal places for all float/double values. (I.e. text/bar scales, anisotropy filter and the sizer). I removed the ability to customize this format from game strings, which required me to hardcode the 1 decimal point for the medipack count in detailed stats screens.

aredfan reviewed (approved):
The sizer inconsistency is resolved, and having 2 decimal places for some of the options looks nice. 👍 

Thank you @rr-
